client/src/main/java/org/apache/nemo/client/ClientUtils.java

```diff
@@ -47,9 +47,8 @@ private ClientUtils() {
    */
   static void handleOptimizationType(final ControlMessage.DriverToClientMessage message,
                                      final DriverRPCServer driverRPCServer) {
-    switch (message.getOptimizationType()) {
-      case XGBoost:
-        new Thread(() ->
+    if(message.getOptimizationType()==XGBoost){
```

Comment:
Would you please insert some spaces like the following?

`if (message.getOptimizationType()==XGBoost) {`